TestFoundation/TestNSFileManager.swift

```diff
@@ -137,6 +138,26 @@ class TestNSFileManger : XCTestCase {
         }
     }
     
+    func test_setFileAttributes() {
+        let path = "/tmp/testfile"
```

Comment:
we might not want to use this since it would potentially conflict with other tests running on CI